### PR TITLE
fix(@communities): Better error message when failing to import

### LINKED ISSUE #5382: The incorrect error appears if the user entered invalid community private key
# Bug Report

## Steps to reproduce
1. Sign in and in Settings - Advanced - Experimental features activate Communities
2. Go to Chat and click on the bullhorn icon -> Communities -> ... -> Access existing c...
3. Enter 'test' in the field for the Community private key and click 'Import'

#### Expected behavior
The correct error appears
#### Actual behavior
The incorrect error at the bottom of the app appears
image

### Additional Information

- Status desktop version: https://github.com/status-im/status-desktop/pull/5338
- Operating System: macOS Monterey, 12.3 Beta (21E5227a)

### PR DESCRIPTION
**Pull request by alaibe**
community

fixes #5382

### CODE REVIEW

File: src/app_service/service/community/service.nim [hunk 1]
```diff
@@ -903,7 +903,7 @@ QtObject:
       error "Error importing the community: ", msg = e.msg
       # We should apply some notification mechanism on the application level which will deal with errors and
       # notify user about them. Till then we're using this way.
-      self.events.emit(SIGNAL_COMMUNITY_IMPORTED, CommunityArgs(error: e.msg))
+      self.events.emit(SIGNAL_COMMUNITY_IMPORTED, CommunityArgs(error: "The community private key is invalid"))
```
Comment (Khushboo-dev-cpp) [inline src/app_service/service/community/service.nim:906]: is this the only possible reason for error when importing a community?

Comment (alaibe) [reply src/app_service/service/community/service.nim:906]: I could make the message more generic: "Error while importing the community!

BTW i kept the error message in the log
wdyt?

Comment (Khushboo-dev-cpp) [reply src/app_service/service/community/service.nim:906]: I think thats better. 
Error message in log looks okay :)

Comment (alaibe) [reply src/app_service/service/community/service.nim]: Message updated ;)

**Review by Khushboo-dev-cpp — APPROVED** [review]

**Review by jrainville — APPROVED** [review]

**Pull request merged**